README.md

```diff
@@ -286,12 +286,8 @@ signIn.renderEl(
   },
 
   function error(err) {
-    // The widget will handle most types of errors - for example, if the user
-    // enters an invalid password or there are issues authenticating.
-    //
     // This function is invoked with errors the widget cannot recover from:
-    // 1. Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR, OAUTH_ERROR
-    // 2. Uncaught exceptions
```

Comment:
This was too vague, as the widget fails in most uncaught exception cases. Removing this statement from our README as it doesn't apply.

Reply:
1. did you figure out what are `uncaught exceptions` or there are actually no others?
2. how about error in RegistrationController where invokes the globalError callback?

Reply:
> did you figure out what are uncaught exceptions or there are actually no others?
I wan't able to discover any occurrences where an unknown error would slip through and need to be handled via the `globalError`. We're explicit in most code execution paths that we're going to either:
a) Let the Widget handle the error (trigger an `error` event on the model/controller)
b) Use the `callGlobalError` method when we see a `Registration`, `OAuth`, `Config` and `UNSUPPORTED_BROWSER` errors.

Regarding registration, I'll add that to the list of known errors above. Sound good?